packages/bpk-component-badge/src/BpkBadgeV2/BpkBadge.module.scss

```diff
@@ -0,0 +1,67 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../bpk-mixins/index.scss';
+
+.bpk-badge-v2 {
```

Comment:
Given the way this has been created and will hopefully superseed the current version of the badge to make life easier you could just have all that is named here be `bpk-badge` as it won't conflict given the file is localised to just eh V2 component